src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -687,6 +687,24 @@ default TypeElement getOutermostTypeElement(Element e) {
      *          elements.getTypeElement("C")); }
      * </blockquote>
      *
+     * Consistent with the usage of the {@link Override @Override}
+     * annotation, if an interface declares a method
+     * override-equivalent to a {@code public} method of {@link Object
+     * java.lang.Object}, such a method of the interface is regarded
+     * as overriding the corresponding {@code Object} method; for
+     * example:
+     *
+     * <blockquote><pre>
+     * interface I {
+     *   &commat;Override
+     *   String toString();
+     * }
+     * ...
+     * assert elements.overrides(elementForItoString,
+     *                           elementForObjecttoString,
+     *                           elements.getTypeElement("I"));
+     * </pre></blockquote>
```

Comment:
Is there a way to have "@Override" represented in the snippet? I can give that a try.

I don't think there is a strong need for this particular code and to be cut-and-pasted easily, but I can use more current features if possible.

Reply:
Snippet works; change pushed. Thanks.

Reply:
Annotations in snippets work as expected. A medium-while back we changed the rules for `@` inside inline tags so that `@` inside an inline tag does not prematurely terminate the inline tag and start a block tag. Thus, `@` can be used for annotations inside `{@snippet...}, `{@code ...}` and any other inline tag.

Aside, FWIW, as a result of seeing your initial version, I did a repo-wide search for legacy use of `&commat;` (used a long time ago to avoid the annotation/block-tag conflict.).  The good news is that there are no other instances of `&commat;` in OpenJDK, so we must have done a cleanup in times past.

Reply:
While snippets do provide the ability to copy the code to the clipboard, they also provide stylistic consistency for the reader, which is good. And maybe one of these days, we'll add support for syntax highlighting ;-)

Reply:
> Aside, FWIW, as a result of seeing your initial version, I did a repo-wide search for legacy use of `&commat;` (used a long time ago to avoid the annotation/block-tag conflict.). The good news is that there are no other instances of `&commat;` in OpenJDK, so we must have done a cleanup in times past.

(drive by comment) One would also need to also look for `&#64;` which is still found in 15 files.

Reply:
@archiecobbs Thanks for that; I'll file an JBS issue to fix them

Reply:
I count 55 instances in 15 files in 5 modules, although not all are in public APIs